homeassistant/components/fastdotcom/__init__.py

```diff
@@ -73,23 +68,7 @@ async def update(service_call: ServiceCall | None = None) -> None:
 
 async def async_unload_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     """Unload Fast.com config entry."""
+    hass.services.async_remove(DOMAIN, "speedtest")
```

Comment:
Nowadays we don't want to remove services when unloading a config entry. The service should be registered in `async_setup` and the service should handle a config entry not being loaded. Look at the fully_kiosk integration for an example.